features/other_a/languages.feature

```diff
@@ -1,7 +1,7 @@
 @works
 Feature: Languages
     
-  Scenario: Browse works by language
+  # Scenario: Browse works by language
```

Comment:
Cucumber now fails on undefined steps (scenario, when, etc.) I think that's the right call, but it means that some TODO tests broke the pipeline

Reply:
~What are the odds we are ever going to do these and wouldn't be better off removing them entirely.~

...who said that?